cmd/clusterCreate.go

```diff
@@ -37,8 +37,11 @@ to quickly create a Cobra application.`,
 	PreRunE: validateClusterCreateFlags,
 	Run: func(cmd *cobra.Command, args []string) {
 
-		nodeCount, _ := cmd.Flags().GetInt("nodes")
-		workerCount := nodeCount - 1
+		workerCount, _ := cmd.Flags().GetInt("worker-count")
+		masterCount, _ := cmd.Flags().GetInt("master-count")
+		etcdCount, _ := cmd.Flags().GetInt("etcd-count")
+		haEnabled, _ := cmd.Flags().GetBool("ha-enabled")
+		isolatedEtcd, _ := cmd.Flags().GetBool("iso-etcd")
```

Comment:
isolated-etcd ?

Reply:
I thought it is unhandy to type a word of this length...thing a full-length is better?

Reply:
I usually see projects like etcd or kubernetes using short version of some words, and later on, switch back to full length.
I think it is better to avoid confusion. When I see short version, sometimes, I always need to go and check the documentation to see to what it refers.
You could also use best of both worlds?

```
-i
--isolated-etcd
```

Just an idea